code/modules/mob/living/carbon/human/human.dm

```diff
@@ -726,4 +726,34 @@
 	staticOverlay.override = 1
 	staticOverlays["letter"] = staticOverlay
 
+mob/living/carbon/human/throw_impact(A)
+	..()
+
+	if(A)
+		if(istype(A, /mob/living/carbon/human)) //Humans can't tackle aliens and borgs

```

Comment:
If you have a decent suggestion on how to let you tackle monkies but not alien or hulks, I'm all for it. Some of the checks aren't brilliant (like the knock-away one)